tsl/test/sql/CMakeLists.txt

```diff
@@ -114,6 +115,7 @@ set(SOLO_TESTS
     cagg_ddl_dist_ht
     cagg_dump
     data_fetcher
+    dist_param
```

Comment:
Why does this need to be a solo test?

Reply:
I was trying to debug it on windows, and I still have no idea what's wrong with it... Sometimes distributed queries just freeze and time out. I guess I'll have to disable it.